pages/series/_itemId/index.vue

```diff
@@ -32,19 +32,10 @@
               'ml-1': $vuetify.breakpoint.mdAndUp
             }"
           >
-            <v-btn
-              v-if="canPlay(item)"
```

Comment:
canPlay can be removed

Reply:
It's part of itemsHelper, which is necessary for other things at the moment (namely the appearances).